src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/Interfaces/ISplitFileService.cs

```diff
@@ -8,8 +8,11 @@ namespace GovUk.Education.ExploreEducationStatistics.Data.Processor.Services.Int
     public interface ISplitFileService
     {
         Task SplitDataFile(
-            ICollector<ImportMessage> collector,
             ImportMessage message,
             SubjectData subjectData);
+
+        Task CreateDataFileProcessingMessages(
```

Comment:
`AddBatchDataFileMessage` might be a better name?

Reply:
Agreed and done